LANGS.md

```diff
@@ -1 +1,2 @@
-* [English](en/)
\ No newline at end of file
+* [English](en/)
+* [Français](fr/)
```

Comment:
I've just seen the rest of this discussion about the language, but I maintain my question :)